churner/churner.go

```diff
@@ -184,7 +184,7 @@ func (c *churner) getOperatorsToChurn(ctx context.Context, quorumIDs []uint8, op
 			return nil, errors.New("maxOperatorCount is 0")
 		}
 
-		if uint32(len(operatorStakes[i])) < operatorSetParams.MaxOperatorCount {
```

Comment:
possibly a bug in previous implementation.
`operatorStakes` should be indexed by quorum ID instead of index of `quorumIDs`